packages/legacy-workbench/src/css/ttyg/agent-settings-modal.css

```diff
@@ -38,12 +38,53 @@
     color: var(--primary-color-dark);
 }
 
-.agent-settings-modal .agent-settings-form .provider-name {
-    padding-left: 5px;
+.agent-settings-modal .agent-settings-form .context-size .input-with-unit {
+    position: relative;
+}
+
+.agent-settings-modal .agent-settings-form .context-size .input-with-unit input {
+    padding-right: 40px;
+}
+
+.agent-settings-modal .agent-settings-form .context-size .input-with-unit .token-unit-label {
```

Comment:
Adding to the above comment, this is not the best approach for styling such field with text inside. Also I think that  using this class `input-group-addon` from bootstrap breaks some stuff here.